package/MDAnalysis/coordinates/PDB.py

```diff
@@ -624,6 +627,9 @@ def __init__(self, filename, bonds="conect", n_atoms=None, start=0, step=1,
            ``False``: write a single frame to the file; ``True``: create a
            multi frame PDB file in which frames are written as MODEL_ ... ENDMDL_
            records. If ``None``, then the class default is chosen.    [``None``]
+        reindex: bool (optional)
+            If ``True`` (default), the atom serial is set to be consecutive
```

Comment:
1. Which value preserves compatibility with 1.x?
2. Have we ever been able to rountrip a PDB when the serials were not consecutively numbered starting from 1? Is this something we would want to be able to do? Is this something a user would generally expect?